cpp/include/cudf/unary.hpp

```diff
@@ -50,14 +51,19 @@ namespace cudf {
  */
 template <typename Fixed,
           typename Floating,
-          typename cuda::std::enable_if_t<is_fixed_point<Fixed>() &&
-                                          cuda::std::is_floating_point_v<Floating>>* = nullptr>
+          CUDF_ENABLE_IF(cuda::std::is_floating_point_v<Floating>&& is_fixed_point<Fixed>())>
```

Comment:
Odd that clang-format didn't change this...
```suggestion
          CUDF_ENABLE_IF(cuda::std::is_floating_point_v<Floating> && is_fixed_point<Fixed>())>
```

Reply:
Whoa, actually it's clang format that is FORCING these.  I remove the spaces and clang-format adds them back.  Very weird.

Reply:
Even if add a newline between the '>' and the '&&' it restores it like this.

Reply:
This is probably a bug in clang-format. We do see those from time to time. I would not worry about it, we will eventually update our clang-format pinnings across RAPIDS and perhaps it will resolve itself.